docs/config.json

```diff
@@ -741,9 +773,9 @@
     {
       "timeline": [
         {
-          "moved": 0,
-          "ringId": "assess",
-          "date": "2022-05-01",
+          "moved": 1,
+          "ringId": "trial",
+          "date": "2023-04-01",
           "description": ""
         }
```

Comment:
minor, but I believe the intention is to keep adding to the timeline, versus updating it. i.e. would look like this:

```
"timeline": [
  {
    "moved": 0,
    "ringId": "assess",
    "date": "2022-05-01",
    "description": ""
  },
  {
     "moved": 1,
     "ringId": "trial",
     "date": "2023-04-01",
     "description": "Node is now being used in production: Recording Proxy Service"
   }
```

Reply:
ahhh. I can make another PR to fix it.